sources/views/activitiesForm.js

```diff
@@ -43,15 +43,19 @@ export default class ActivitiesForm extends JetView {
 		this.form = this.$$("activitiesForm");
 	}
 
-	showForm(id) {
-		if (id && activities.exists(id)) {
+	showForm(id, add) {
+		if (id && !add && activities.exists(id)) {
 			const item = activities.getItem(id);
 			this.form.setValues(item);
 		}
+		if (id && add && contacts.exists(id)) {
+			this.$$("combo").setValue(id);
+			this.$$("combo").define("readonly", true);
```

Comment:
it is disabled in code and when click on combo, but add some css to define combo readonly status

Reply:
Fixed